### PR TITLE
Feature/11.2.x 12.1.x fixes

### PR DESCRIPTION
**Pull request by lramarojaona**

### CODE REVIEW

File: scripts/composer/OpenfedUpdate.php
Comment (rutiolma) [inline scripts/composer/OpenfedUpdate.php:1]: to review

File: scripts/composer/OpenfedValidations.php
Comment (rutiolma) [inline scripts/composer/OpenfedValidations.php:1]: to review

**Review by rutiolma — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
 CHANGELOG
 =========
 
+06 September 2023 - Version 11.2.0
```
[the remark this answers is not in the document]
Comment (lramarojaona) [reply CHANGELOG.md:10]: I've added the changelog for 12.1.0

File: composer.json [hunk 2]
```diff
@@ -128,9 +128,12 @@
                 "Issue #2870738: Wrong active trail with multiple menu items with the same path": "https://www.drupal.org/files/issues/2022-03-22/multiple-menu-links-2870738-20.patch",
                 "Issue #2914746: Fatal error after installing the module (incompatibility with Context & Menu position module)": "https://www.drupal.org/files/issues/2022-04-06/2914746-27.patch"
             },
+            "drupal/onomasticon": {
+                "Issue #3340245: Enhance accessibility on Onomasticon module": "https://www.drupal.org/files/issues/2023-09-22/onomasticon-enhance-accessibility-3340245-7.patch"
+            },
             "drupal/page_manager": {
                 "Issue #2820218: Page manager does not respect existing route defaults for title callbacks": "https://www.drupal.org/files/issues/2018-03-21/2820218-50.patch",
-                "Issue #2858877: Allow for block visibility rules": "https://www.drupal.org/files/issues/2023-05-09/page_manager-block_visibility_conditions-2858877-75.patch"
+                "Issue #2858877: Allow for block visibility rules": "https://www.drupal.org/files/issues/2022-11-15/page_manager-block_visibility_conditions-2858877-71.patch"
```
[the remark this answers is not in the document]
Comment (lramarojaona) [reply composer.json]: I updated the patch with the last version

File: composer.json [hunk 3]
```diff
@@ -285,7 +289,7 @@
         "drupal/tmgmt": "1.15",
         "drupal/token": "1.11",
         "drupal/twig_field_value": "2.0.2",
-        "drupal/twig_tweak": "3.2.1",
+        "drupal/twig_tweak": "2.10",
```
[the remark this answers is not in the document]
Comment (lramarojaona) [reply composer.json]: Fixed

File: openfed.info.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@ name: Openfed
 type: profile
 description: 'OpenFed base installation with modules and configurations needed for Fast2Web offer by Fedict.'
 core_version_requirement: ^9
-version: '11.2.0-beta1'
+version: '11.2.0'
```
[the remark this answers is not in the document]
Comment (lramarojaona) [reply openfed.info.yml]: New version set to 12.1.0

File: openfed.install [hunk 5]
```diff
@@ -80,3 +80,67 @@ function openfed_update_91103() {
     }
   }
 }
+
+/**
+ * Fix a template regression issue with office_hours module.
```
[the remark this answers is not in the document]
Comment (lramarojaona) [reply openfed.install]: This has been removed.

**Review by rutiolma — APPROVED** [review]

**Pull request merged**